kqueen/blueprints/api/generic_views.py

```diff
@@ -140,15 +148,34 @@ class ListView(GenericView):
     action = 'list'
 
     def set_object(self, *args, **kwargs):
+        if request.args.get('all_namespaces'):
```

Comment:
Why `args`? I'd expect it to be `kwarg` - `namespace='all'`

Reply:
List is a GET only endpoint, so the `all_namespaces` param is passed via urlencode and appears as arg on request